src/plugins/blocksmith/components/blocksmith/Blocksmith.jsx

```diff
@@ -96,7 +93,6 @@ import {
   handleDocumentDrop,
   insertCanvasBlocks,
   insertEmptyBlock,
-  isAtomicBlockSelected,
```

Comment:
@tbone1020 i think this is also not used anywhere anymore eh? if so delete it

Reply:
done